climakitae/data_export.py

```diff
@@ -239,27 +218,71 @@ def _export_to_user(user_export_format, data_to_export,
     if ("NetCDF" in req_format):
         export_to_netcdf(data_to_export, save_name, **kwargs) 
     else:
-        assert "DataArray" in str(type(data_to_export)), ("We are only"
-                             + " converting data arrays to this format"
-                             + " at this time, please pass a data array"
-                             + " (not a dataset). HINT:" )
+        if "DataArray" not in str(type(data_to_export)):
```

Comment:
`.data_vars` is a `Dataset` property, not a `DataArray` property. a `Dataset` can have multiple data variables. this condition is encountered after the `xarray` object check has passed, so we know it's either a `Dataset` or `DataArray`. 

why am I using a `not`? because I swear that `type(DataArray)` returned `xarray.core.dataset.DataArray`, so using `if "Dataset" in....` as the conditional would not work... but now I'm seeing  `xarray.core.dataarray.DataArray`. 

I think that perhaps I wasn't using `pip install requirements.txt`, so I might have been using the wrong xarray version before. next commit will check for `Dataset` rather than `not DataArray`.